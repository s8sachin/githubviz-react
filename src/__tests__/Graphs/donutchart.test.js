import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount, render, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Donutcharts from '../../components/Graphs/donutchart';

configure({ adapter: new Adapter() });

describe('Donutcharts', () => {
it('should mount donutchart component', () => {
    const wrapper = shallow(<Donutcharts />)
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})