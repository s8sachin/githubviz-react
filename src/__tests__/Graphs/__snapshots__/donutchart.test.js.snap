// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Donutcharts should mount Header component 1`] = `
<RadialChart
  className="donut-chart-example"
  colorRange={
    Array [
      "#12939A",
      "#79C7E3",
      "#1A3177",
      "#FF9833",
      "#EF5D28",
    ]
  }
  colorType="category"
  data={
    Array [
      Object {
        "className": "custom-class",
        "theta": 11,
      },
      Object {
        "theta": 12,
      },
      Object {
        "theta": 13,
      },
      Object {
        "theta": 14,
      },
      Object {
        "theta": 15,
      },
    ]
  }
  getAngle={[Function]}
  getAngle0={[Function]}
  getLabel={[Function]}
  getRadius={[Function]}
  getRadius0={[Function]}
  getSubLabel={[Function]}
  height={300}
  innerRadius={100}
  onSeriesMouseOut={[Function]}
  onValueMouseOver={[Function]}
  radius={140}
  width={300}
/>
`;
